R/new_matrixfit.R

```diff
@@ -488,6 +520,9 @@ new_matrixfit <- function(cf,
   
   if (model == 'single') {
     model_object <- SingleModel$new(cf$Time, parlist, sym.vec, neg.vec, mSize)
+  } else if (model == 'two_amplitudes') {
```

Comment:
doesn't this need to be mentioned in the roxygen docu as well?

Reply:
Yes, it should be mentioned in the documentation. I have added formulas for most models.

Reply:
The weighted model is way more complicated:

```r
pred <- ov_sign_vec * (a0*(-((exp(2*e0*time) + exp(e0*(2*t + time))*sign)*Power(w,2*t)) - sign*(exp(2*e0*time) + exp(e0*(2*t + time))*sign)*Power(w,2*deltat + time) + (exp(e0*(deltat + 2*time)) + exp(e0*(-deltat + 2*t + time))*sign)*Power(w,deltat)* (Power(w,2*t) + sign*Power(w,time)))) / (exp(e0*(t + 2*time))*Power(w,deltat)*(Power(w,2*t) + sign*Power(w,time)))
```

I am not sure whether one could actually read it. I've derived it with Mathematica and just copied it out of there.